scripts/write_deployinst_input.py

```diff
@@ -380,28 +419,21 @@ def region_render(list, template, full_template, output_file):
         os.system('rm ' + country + '_region')
 
 
-def main(csv_file, init_date, duration, reactor_template, mox_reactor_template,
-         reprocessing, deployinst_template, input_template, output_file):
+def main(csv_file, init_date, duration, output_file, reprocessing=True):
     """ Generates cyclus input file from csv files and jinja templates.
 
     Parameters
     ---------
     csv_file : str
         csv file containing reactor data (country, name, capacity)
     init_date: int
-        yyyymmdd format of inital date of simulation
-    reactor_template: str
-        template file for reactor section of input file
-    mox_reactor_templtae: str
-        template file for mox reactor section of input file
-    reprocessing: bool
-        True if reprocessing is done, False if not
-    deployinst_template: str
-        template file for deployinst section of input file
+        yyyymmdd format of initial date of simulation
```

Comment:
If you're going to collect a yyyymmdd, collect three parameters, year, month, day. OR collect a three-tuple (y,m,d) or even better, a dictionary {"year":yyyy,"month":mm, "day":dd

Requiring a format like this is buggy and requires computationally unnecessary unpacking.

Reply:
That is a good idea. I'm not sure how the user end would look like though, because the way to make this is either
1. the user types in a three-tuple or dictionary
2. the conversion from yyyymmdd is done in a function ( which is sort of what `get_ymd` is doing now)
What do you think?

Reply:
The user can just input three variables.